charts/ingresslibchart/templates/_ingress.yaml

```diff
@@ -0,0 +1,23 @@
+{{- define "libchart.ingress.tpl" -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ .Chart.Name }}-ingress
+  labels:
+    {{- include "helm.labels" . | nindent 4 }}
+  annotations:
+    networking.gke.io/v1beta1.FrontendConfig: "{{ .Chart.Name }}-ingress-frontendconfig"
+    kubernetes.io/ingress.global-static-ip-name: {{ required "ingress.staticIpName value is required" .Values.ingress.staticIpName | quote }}
```

Comment:
I don't know if it makes sense here, but documenting the expected values file structure might be helpful. I know a values file won't be included as defaults or anything because it's a library chart but maybe even an example in a readme or something would help

Reply:
Right now, we have no value files structure. However, when looking at how other applications define ingress values, we seem to be aligned. The error message "ingress.staticIpName" says where the value should be located.